tests/benchmark.py

```diff
@@ -125,6 +125,7 @@ def generate_test_configs(input_path: str, test_attributes: List[str]) -> List[D
             "gpt-4o",
             "gpt-4o-mini",
             "meta-llama/Llama-3.2-11B-Vision-Instruct",
+            "meta-llama/Llama-3.2-90B-Vision-Instruct-Turbo",
```

Comment:
For me
`f"{key}={str(value).replace("/", "_")}"` is throwing an error. I tried escaping but that didn't work either not sure why.
`f"{key}={str(value).replace('/', '_')}" ` --> works